pipelines/LsstCamImSim/ApPipeCalibrateImage.yaml

```diff
@@ -0,0 +1,10 @@
+# TODO DM-40388: This file will replace ApPipe.yaml, once the new pipeline is vetted.
+description: End to end AP pipeline specialized for ImSim.
+# (1) Execute `make_apdb.py`, e.g.,
+#     make_apdb.py -c db_url="sqlite:////project/user/association.db"
+# (2) Run this pipeline, setting appropriate diaPipe configs
+#     (diaPipe configs should match the make_apdb.py configs)
+
+instrument: lsst.obs.lsst.LsstCamImSim
+imports:
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipeCalibrateImage.yaml
```

Comment:
That seems a bit odd, since `_ingredients/ProcessCcdCalibrateImage.yaml` is already included by `_ingredients/ApPipeCalibrateImage.yaml`. Do we actually want that duplication in the instrument pipelines, or should we only add such if we know we need it (as in the current characterize/calibrate overrides)?

Reply:
Yes, it's redundant, but it would let you implement DM-40388 as a one-line substitution. The current `ApPipeCalibrateImage.yaml` is built around the fact that it _doesn't_ do any splicing, so adding it in later would be a major rewrite.

Reply:
Oh, I see what you meant -- "even if the new `ProcessCcd.yaml` is a no-change"? I think we do, just because the assumption that `<instrument>/ProcessCcd.yaml` doesn't have any tweaks is a brittle one.

Reply:
Wouldn't most such tweaks also require us to add subtask redefinitions to the instrument files anyway?

Reply:
I'm not sure I follow the argument; I was thinking of things like different configs. But maybe in the post-RFC-775 world, configs that need to be both AP- and instrument-specific are rarer?

Reply:
I was thinking of things like the DECam overrides that exclude isr and thus require redefining the `processCcd` subset in `ApPipe.yaml`.

But yes, once DM-39796 is merged, I'd expect instrument-specific AP overrides to be very rare, and be almost exclusively in ISR (as in DECam).